CommerceBureau.h

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+#include "LeakedObjectDetector.h"
+#include "ShelterOfCats.h"
+#include "Safeway.h"
+#pragma once
```

Comment:
`#pragma once` should be the first line written in this header file
- [ ] revise